readme.md

```diff
@@ -154,7 +154,21 @@ Defer showing the notification to after the process has exited.
 Type: `string`\
 Default: [See above screenshot](https://github.com/yeoman/update-notifier#update-notifier-)
 
-Message that will be shown when an update is available.
+Message that will be shown when an update is available, support [`pupa`](https://github.com/sindresorhus/pupa) template.
```

Comment:
I don't think we need to mention `pupa`. Can just say that it supports placeholders that will be replaced.